released/packages/coq-sail/coq-sail.0.17.1/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: "Sail Devs <cl-sail-dev@lists.cam.ac.uk>"
+authors: [
+  "Alasdair Armstrong"
+  "Thomas Bauereiss"
+  "Brian Campbell"
+  "Shaked Flur"
+  "Jonathan French"
+  "Kathy Gray"
+  "Robert Norton"
+  "Christopher Pulte"
+  "Peter Sewell"
+  "Mark Wassell"
+]
+homepage: "http://www.cl.cam.ac.uk/~pes20/sail/"
+bug-reports: "https://github.com/rems-project/coq-sail/issues"
+doc: "https://github.com/rems-project/sail/blob/sail2/README.md"
+license: "BSD-3-clause"
+dev-repo: "git+https://github.com/rems-project/coq-sail.git"
+build: [make "-C" "src" "HAVE_OPAM_BBV=yes"]
+install: [make "-C" "src" "install"]
+depends: [
+  "coq" {>= "8.14.0" & <= "8.18.0"}
```

Comment:
```suggestion
  "coq" {>= "8.14.0" & < "8.19"}
```
Note that the Coq release manager ensures that Coq minor versions, such as the planned 8.18.1, will be compatible with any Coq project supporting 8.18.0. This is why we recommend support all minor versions of a major version. Without this change, this package won't allow 8.18.1.